lib/stat_tracker.rb

```diff
@@ -12,11 +12,11 @@ def self.from_csv(locations)
     @stat_tracker = StatTracker.new(@data_array)
   end
 
-  def initialize(data_array)
+  def initialize(data_array, games = nil, teams = nil, game_teams = nil)
```

Comment:
I added these default arguments so I could bypass the CSV reading.  To get sophisticated test data across three files became too complicated so I made the data for the test in line